Dockerfile

```diff
@@ -14,15 +14,14 @@ RUN \
 
 WORKDIR /build
 
-ENV VERSION 1.39.1
+ENV VERSION 1.40.0
 
 ADD https://github.com/fastly/pushpin/releases/download/v${VERSION}/pushpin-${VERSION}.tar.bz2 .
 
 RUN tar xf pushpin-${VERSION}.tar.bz2 && mv pushpin-${VERSION} pushpin
 
 WORKDIR /build/pushpin
 
-RUN cargo fetch
 RUN make RELEASE=1 PREFIX=/usr CONFIGDIR=/etc
```

Comment:
it would be nice to combine RUN command to minimize layers and image size. sth like this
```
RUN make RELEASE=1 PREFIX=/usr CONFIGDIR=/etc && \
    make RELEASE=1 PREFIX=/usr CONFIGDIR=/etc check && \
    make RELEASE=1 PREFIX=/usr CONFIGDIR=/etc INSTALL_ROOT=/build/out install
```